docs/reference/how-to/use-elasticsearch-for-time-series-data.asciidoc

```diff
@@ -0,0 +1,234 @@
+[[use-elasticsearch-for-time-series-data]]
+== Use {es} for time series data
+
+{es} offers features to help you store, manage, and search time series data,
+such as logs and metrics. Once in {es}, you can analyze and visualize your data
+using {kib} and other {stack} features.
+
+To get the most out of your time series data in {es}, follow these steps:
+
+* <<set-up-data-tiers>>
+* <<register-snapshot-repository>>
+* <<create-snapshot-lifecycle-policy>>
+* <<create-edit-index-lifecycle-policy>>
+* <<create-ts-component-templates>>
+* <<create-ts-index-template>>
+* <<search-visualize-your-data>>
+
+
+[discrete]
+[[set-up-data-tiers]]
+=== Step 1. Set up data tiers
+
+{es}'s <<index-lifecycle-management,{ilm-init}>> feature uses <<data-tiers,data
+tiers>> to automatically move older data to nodes with less expensive hardware
+as it ages. This helps improve performance and reduce storage costs.
+
+We recommend setting up at least a hot tier for your cluster. Add warm, cold,
+and frozen tiers based on your performance and storage needs.
+
+Use high performance nodes in the hot and warm tiers for faster indexing and
+faster searches on your most recent data. Use slower, less expensive nodes in
+the cold and frozen tiers to reduce costs.
+
+The steps for setting up data tiers vary based on your deployment type:
+
+include::{es-repo-dir}/tab-widgets/code.asciidoc[]
+include::{es-repo-dir}/tab-widgets/data-tiers-widget.asciidoc[]
+
+[discrete]
+[[register-snapshot-repository]]
+=== Step 2. Register a snapshot repository
+
+A <<snapshot-restore,snapshot>> backs up your cluster and its data. The cold and
+frozen tiers uses <<searchable-snapshots,searchable snapshots>> to reduce local
+storage costs.
+
+To create snapshots, you must register a snapshot repository. The repository is
+typically a remote object store, such as Google Cloud Storage (GCS) or Azure
+Blob Storage.
+
+The steps for registering a snapshot repository vary based on your deployment
+type and storage provider:
+
+include::{es-repo-dir}/tab-widgets/snapshot-repo-widget.asciidoc[]
+
+[discrete]
+[[create-snapshot-lifecycle-policy]]
+=== Step 3. Create a snapshot lifecycle policy
+
+<<snapshot-lifecycle-management,{slm-init}>> uses a snapshot lifecycle policy to
+automate the creation and deletion of snapshots. This ensures your data is
+backed up on a regular basis.
+
+To create a snapshot lifecycle policy in {kib}, open the main menu and go to
+**Stack Management > Snapshot and Restore > Policies**. Click **Create Policy**.
+
+You can also use the <<slm-api-put-policy,create snapshot lifecycle policy
+API>>.
+
+[source,console]
+----
+PUT /_slm/policy/my-snapshot-policy
+{
+  "schedule": "0 30 2 * * ?",
+  "name": "<nightly-snapshot-{now/d}>",
+  "repository": "my-snapshot-repository",
+  "retention": {
+    "expire_after": "750d",
+    "min_count": 5,
+    "max_count": 800
+  }
```

Comment:
@ywelsch Let me know if this retention configuration seems realistic. `800` seems high, but our ILM example keeps data around for ~ 2 years.

Reply:
ILM uses its own snapshots for snapshot-backed indices, which are not subject to the retention from SLM. We currently suggest < 200 (see https://github.com/elastic/elasticsearch/issues/66042 for some background), so I don't think we should go as high as 800, or as long as 750d

Reply:
Thanks @dakrone. I wasn't aware that ILM used its own snapshots. I'll remove the SLM section entirely. While it's still a good idea, it's not really about time series stuff.